static/app/views/performance/transactionSummary/transactionSpans/content.tsx

```diff
@@ -29,6 +30,23 @@ import SuspectSpanCard from './suspectSpanCard';
 import {SpansTotalValues} from './types';
 import {getSuspectSpanSortFromEventView, SPAN_SORT_OPTIONS} from './utils';
 
+const ANALYTICS_VALUES = {
+  spanOp: (organization: Organization, op: string | null) =>
+    trackAnalyticsEvent({
```

Comment:
@Zylphrex `trackAnalyticsEvent()` is deprecated. Can you use `trackAdvancedAnalyticsEvent()` like how it's used in https://github.com/getsentry/sentry/pull/30240 ?